libs/ui-toolkit/modals/src/public_api.ts

```diff
@@ -0,0 +1,7 @@
+export * from './lib/ui-modal/ui-modal.module';
+
+export * from './lib/ui-modal/ui-simple-modal/ui-simple-modal.component';
+export * from './lib/ui-modal/ui-loading-modal/ui-loading-modal.component';
+
+
+
```

Comment:
Check all files and remove extra empty lines (they were probably automatically added by prettier, so pay attention when committing to not include them in your diffs)